backend/app/api/diagnosis.py

```
@@ -31,6 +36,51 @@ class CreateChatMessageRequest(BaseModel):
 router = APIRouter(prefix="/diagnosis", tags=["diagnosis"])
 logger = logging.getLogger(__name__)
 
+# 💡 이미지 저장을 위한 디렉토리 경로 설정
+UPLOAD_DIR = Path("data/uploads/diagnosis")
+UPLOAD_DIR.mkdir(parents=True, exist_ok=True)
+
+@router.post("/upload")
+async def upload_diagnosis_image(
+    file: UploadFile = File(...),
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)